src/_blog/cryptotls.md

```diff
@@ -0,0 +1,58 @@
+
+This means that crypto/tls needed to support adding data to session tickets, both the client and server sides. It turned out that the API required for this would also solve a large number of other longstanding [issues related to session tickets](https://github.com/golang/go/issues/60105).
+
+This is only one of the problems we had to solve to make 0-RTT work. Another complication arises from the fact that the server can reject 0-RTT for any reason (typically DoS protection or because it doesn’t consider the QUIC transport parameters acceptable anymore). In that case, 0-RTT packets are discarded and the handshake continues as a regular (1-RTT) QUIC handshake. This necessitated a new [API for deciding about 0-RTT rejection](https://github.com/golang/go/issues/60107) and for clients to be informed of it.
+
+## Current Status
+
+After an intense period of collaboration and development, we are thrilled to have all these changes included in the Go 1.21 release. The implementation of these proposals in the standard library was performed by Damien Neil and Filippo Valsorda. On the quic-go side, we made use of the new APIs as soon as the changes were published. This allowed us to use quic-go's test suite to stress-test the implementation. This was really valuable, since quic-go’s test suite is quite extensive and covers a large number of corner cases. And it indeed found a small issue, which we subsequently [fixed](https://go-review.googlesource.com/c/go/+/498215).
+
+And while we have a small workaround for one remaining [issue](https://github.com/golang/go/issues/60506), we are hopeful of a resolution in the next release.
+
+The newly introduced changes have been released in quic-go v0.37 (TODO: link), and in go-libp2p (TODO: insert version). We anticipate that these changes will work seamlessly once users update their dependencies and compiler version, but please let us know if you run into any problems!
+
+## A few words on the Go team’s QUIC efforts
+
+It looks like the Go team has started working on [their own QUIC implementation](https://github.com/golang/go/issues/58547). As we know from the 8-year journey implementing quic-go, writing a performant QUIC stack is a massive undertaking. And we’re not done yet, we’re still investing significant resources into optimizing it. 
+
+We have [offered](https://github.com/golang/go/issues/58547#issuecomment-1463211376) to use quic-go as a basis for the standard library implementation, and even made a [concrete proposal](https://github.com/golang/go/issues/58547#issuecomment-1569583646) how it could be used to add HTTP/3 support to the net/http package. Unfortunately, the Go team has never reached out to us, or taken up our proposal to jump on a call to evaluate the feasibility of this approach.
```

Comment:
This sounds like you're accusing the Go team of being mean/not doing their job. I'm not sure it is wise to put this kind of pressure on them.

Reply:
I removed the entire section.